fix open_pdks.gf180mcuc build failure

See https://github.com/hdl/conda-eda/actions/runs/5127108900/jobs/9222456725

```
cp: cannot stat '/root/conda-eda/conda-eda/workdir/conda-env/conda-bld/open_pdks.gf180mcuc_1685489184112/work/gf180mcu-pdk/gf180mcu_fd_pr/rules/klayout/lvs/*': No such file or directory
make[2]: *** [Makefile:719: klayout-C] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [Makefile:634: all-C] Error 2
make: *** [Makefile:132: tech-gf180mcu] Error 2
```

Seems to be related to some directory layout change in the physical verication part of the PDK.

--- a/misc/open_pdks/gf180mcuc/fix-klayout-lvs-path.patch
+++ b/misc/open_pdks/gf180mcuc/fix-klayout-lvs-path.patch
@@ -1,0 +1,13 @@
+diff --git a/gf180mcu/Makefile.in b/gf180mcu/Makefile.in
+index 5a71821..2842a03 100644
+--- a/gf180mcu/Makefile.in
++++ b/gf180mcu/Makefile.in
+@@ -718,7 +718,7 @@ klayout-%: ${GF180MCU_PV_PATH} ${GF180MCU_PR_PATH}
+ 
+ 	cp -rp ${GF180MCU_PV_PATH}/klayout/drc/* \
+ 		${KLAYOUT_STAGING_$*}/drc
+-	cp -rp ${GF180MCU_PR_PATH}/rules/klayout/lvs/* \
++	cp -rp ${GF180MCU_PV_PATH}/klayout/lvs/* \
+ 		${KLAYOUT_STAGING_$*}/lvs
+ 	cp -rp ${GF180MCU_PR_PATH}/cells/klayout/pymacros/cells/* \
+ 		${KLAYOUT_STAGING_$*}/pymacros

--- a/misc/open_pdks/gf180mcuc/meta.yaml
+++ b/misc/open_pdks/gf180mcuc/meta.yaml
@@ -11,6 +11,7 @@ source:
     patches:
       - use-sources-from-conda.patch
       - disable-link-targets-for-C-variant.patch
+      - fix-klayout-lvs-path.patch
   - git_url: https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pr.git
     git_rev: main
     git_depth: 1

